apps/examples/st_things/light/sample_light_user.c

```diff
@@ -36,6 +36,106 @@ static bool g_switch_value = false;
 static int64_t g_dimming_setting = 50;
 static int64_t g_color_temp = 50;
 
+#define EXAMPLE 1
+
```

Comment:
is this the right place to declear the EXAMPLE?

Reply:
please remove this commit, 
we need to add another TC app